components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/internal/DCRDataHolder.java

```diff
@@ -33,7 +34,6 @@
  * This was deprecated as part of deprecating the legacy identity/register DCR endpoint.
  * The recommendation is to use /identity/oauth2/dcr/v1.1 instead.
  */
-@Deprecated
```

Comment:
Why we are removing this flag? I also wants to know why this flag introduced in the first place. If you have any hint please state.